bootstrap/kubeadm/internal/bottlerocket/bottlerocket.go

```diff
@@ -150,6 +156,20 @@ func getBottlerocketNodeUserData(bootstrapContainerUserData []byte, users []boot
 	return bottlerocketNodeUserData, nil
 }
 
+func parseNodeLabels(nodeLabels string) string {
+	if nodeLabels == "" {
+		return ""
+	}
+	nodeLabelsToml := ""
+	for _, nodeLabel := range strings.Split(nodeLabels, ",") {
```

Comment:
can we save the output of `strings.Split(nodeLabels, ",")` before the for loop and then range over it?